profiles/base.yaml

```diff
@@ -234,6 +234,8 @@ meditation_pause_timer: 20
 # You won't try to activate a barb buff until you have this much inner fire.
 # You need some inner fire in reserve to pay the periodic cost of abilities.
 barb_buffs_inner_fire_threshold: 15
+# Minimum inner fire to trigger analyze flame during whirlwinding
```

Comment:
If we change the conditional per my comment at https://github.com/rpherbig/dr-scripts/pull/4791/files#r591940668 then this comment can drop the "_during whirlwinding_"